superset/assets/javascripts/explore/components/ExploreChartHeader.jsx

```diff
@@ -43,13 +43,13 @@ class ExploreChartHeader extends React.PureComponent {
       slice_name: newTitle,
       action: isNewSlice ? 'saveas' : 'overwrite',
     };
-    const saveUrl = getExploreUrl(this.props.form_data, 'base', false, null, params);
-    this.props.actions.saveSlice(saveUrl)
+    this.props.actions.saveSlice(this.props.form_data, params)
       .then((data) => {
         if (isNewSlice) {
           this.props.actions.createNewSlice(
             data.can_add, data.can_download, data.can_overwrite,
             data.slice, data.form_data);
+          this.props.addHistory({ isReplace: true, title: '[slice] ' + data.slice.slice_name });
```

Comment:
nit - template literals? there's inconsistent usage in this PR.

Reply:
fixed. now use template.